src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
Writing CSS can become hard to maintain when the stylesheet gets huge. And in this case, the preprocessor can help us.

CSS preprocessor is a program that has its own syntax but can "translate" CSS code from it. There are many preprocessors. One of them is Sass.

Sass allows using features not available in CSS, such as nesting, mixins, functions, loops, and other useful things.